doc/src/fix_neb.rst

```diff
@@ -124,6 +125,20 @@ in force units.
 Note that the *ideal* form of nudging can often be more effective at
 keeping the replicas equally spaced.
 
+With a value of *equal* the spring force is computed as for *ideal*
+before the climbing stage, then is computed to promote equidistant 
+spacing in energy rather than distance:
+
+.. parsed-literal::
```

Comment:
Mathematical expressions are better typeset with a `.. math::` block or a `:math:` role. We should use the opportunity of this pull request to revise the entire NEB documentation accordingly. I am happy to take a stab at it unless you prefer to do it yourself.

Reply:
I would be very happy to take up the second iteration as I am confident you will be much better aligned in style with the rest of the doc!

Reply:
Ok. Please ping me, when you are done making changes, so I would step on your feet when I grab the branch and make modifications.